app/services/user/import.rb

```diff
@@ -0,0 +1,35 @@
+class User
+  class Import < ApplicationService
+    def initialize(emails:, separator: ',')
+      @separator   = separator.strip
+      @emails = emails
+    end
+
+    def call
+      (prepared_emails - existing_user_emails).each do |email|
+        first_name = name_email_part(email).first
+        last_name = name_email_part(email).last
+
+        user = User.new(email: email, first_name: first_name, last_name: last_name)
```

Comment:
Here if there is no `.` in email, user won't be created, because of last name validation.

Reply:
Ohh, yes, you're right. My bad.